patchutils_test.go

```diff
@@ -257,9 +199,9 @@ func TestInterDiffMode(t *testing.T) {
 			currentResult, err := InterDiff(readerA, readerB)
 
 			if (tt.wantErr == nil) && (err == nil) {
-				if !bytes.Equal(normalizeNewlines([]byte(currentResult)), normalizeNewlines(correctResult)) {
-					t.Errorf("File contents mismatch for %s.\nExpected:\n%s\nGot:\n%s\n",
-						tt.resultFile, correctResult, currentResult)
+				if d := cmp.Diff(normalizeNewlines(correctResult), normalizeNewlines([]byte(currentResult))); d != "" {
```

Comment:
please use `cmp.Equal` for the condition and `cmp.Diff` only in the output in case got != want. Please also fix the cases below accordingly.